containers/init/ready/ready.go

```diff
@@ -57,14 +58,22 @@ const OutputFileEnv = "READY_OUTPUT_FILE"
 // write all metadata.
 const OutputMetadataEnv = "METADATA_OUTPUT_FILE"
 
-// DefaultMetadataOutputFile is the name of the default file where the executable should
-// write the metadata.
-const DefaultMetadataOutputFile = "/tmp/metadata.json"
+// OutputNodeInfoEnv is the optional name of the file where the executable should
```

Comment:
Can I ask why is it optional?

Reply:
I assume those environment variables are all optional, like "OutputMetadataEnv, OutputFileEnv".

Reply:
I think the variable itself may not be used but it is not really optional though. The name it sets for the file is the same with default name, it is a bit confusing  to say it is optional.

Reply:
I am confused here. The comment for `OutputNodeInfoEnv` is rooted from `OutputMetadataEnv` and `OutputFileEnv`. Are you suggesting that we should change them too?

Reply:
No, you are right, diff that github shows lead me think you modified the default to optional. I think it is fine to use optional that all others has been using. I just felt that these ENVs are actually hardcoded in podbuilder, they are the actual default (i.e if we don't do anything)